package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Symphony",
-  "productName": "Symphony",
+  "productName": "Symphony-Electron",
```

Comment:
I think this depends on Symphony's branding strategy for the Electron App and on whether the existing Desktop Wrapper will be continued to be used after the Electron wrapper has been released. We can probably keep this as "Symphony" only.

Same in other places as well.

@lneir Please share your inputs.